challenge.md

```diff
@@ -32,19 +32,19 @@ Open up this file in your text editor, and write your answers below (where indic
 1. Is `/Users/blah_blah/Desktop` a relative path or an absolute path? How do you know?
 
  <!-- Answer Starts Here -->
- 
+ Absolute path because it tells you the location of the desktop relative to the root directory.
  <!-- Answer Ends Here -->
 
 2. Suppose that we're working on a project, and we want to use a font that's being hosted somewhere on the internet. Would we use an absolute or relative path to refer to it? Why?
 
  <!-- Answer Starts Here -->
- 
+ Absolute path, the font being used has one path to it called URL.
  <!-- Answer Ends Here -->
 
 3. Now suppose that we have an image file living inside our project. Would we refer to it with an absolute or relative path? Why?
 
  <!-- Answer Starts Here -->
- 
+ Relative path because the image is not located at a same place for everyone for access.
```

Comment:
Right answer, wrong reason. We would want to refer to our image using its Relative path because it's possible that we might want to move our project folder; if we were to do that, any links that used Absolute paths would break.